README.md

```diff
@@ -119,6 +119,40 @@ query: {
 }
 ```
 
+### $exists
+
+[Term level query `exists`](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-exists-query.html). Find all documents that have at least one non-null value in the original field. The value can be an object or an array (not analyzed). 
+
+```js
+query: {
+  $exists: { field: 'phone' }
```

Comment:
We try to keep the payload as lean as possible. I would suggest the query to accept an array of fields, like so:

`$exists: ['phone', 'address']`